packages/docgen/lib/markdown/formatter.js

```diff
@@ -68,9 +73,26 @@ const formatDeprecated = ( tags, docs ) => {
 };
 
 const formatDescription = ( description, docs ) => {
+	const processor = remark().use( () => {
+		return function transformer( tree ) {
+			tree.children.forEach( function ( node ) {
+				if ( node.children ) {
+					transformer( node );
+				}
+				if ( node.type === 'text' && node.value ) {
+					// Replace line breaks with spaces and remove hyphens from hyphenated words.
```

Comment:
the code is right here, but the comment is misleading the way I interpret it.

> Replace line breaks with spaces and remove line-ending hyphens.

Based on the comment I expected "client-side" to be replaced by "clientside" though it was me being wrong with that assumption. It could help to have an example in the test code that shows hyphens that aren't replaced.

```js
Client-side hyphen- 
ation.
```

```
My declaration example. Client-side hyphenation.
```

Reply:
Thanks for the advice. I have updated with [da8bd19](https://github.com/WordPress/gutenberg/pull/49635/commits/da8bd198373048cd69fc17138bd33728d7efc78f) and [b0af647](https://github.com/WordPress/gutenberg/pull/49635/commits/b0af64702548aa63e953ce9269a8912c289e6ac3).